main.go

```diff
@@ -1,16 +1,26 @@
 package main
 
 import (
+	"crypto/tls"
 	"encoding/json"
 	"fmt"
+	"net/http"
 	"os"
+	"strings"
 
+	cmmeta "github.com/jetstack/cert-manager/pkg/apis/meta/v1"
 	extapi "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
-	//"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 
 	"github.com/jetstack/cert-manager/pkg/acme/webhook/apis/acme/v1alpha1"
 	"github.com/jetstack/cert-manager/pkg/acme/webhook/cmd"
+	"github.com/jetstack/cert-manager/pkg/issuer/acme/dns/util"
+
+	ns1API "gopkg.in/ns1/ns1-go.v2/rest"
+	ns1DNS "gopkg.in/ns1/ns1-go.v2/rest/model/dns"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 var GroupName = os.Getenv("GROUP_NAME")
```

Comment:
I think this can be unexported. Either that or it should have comment

Reply:
yeah this was bugging me too but not sure how it's used. Now that things are working i feel more comfortable seeing if it needs to be exported

Reply:
unexported that puppy